pkg/network/podinterfacenamescheme/podinterfacenamescheme.go

```diff
@@ -19,4 +19,37 @@
 
 package podinterfacenamescheme
 
+import (
+	"fmt"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/network/multus"
+)
+
 const PrimaryPodInterfaceName = "eth0"
+
+type PodInterfaceNamer map[string]string
+
+func NewPodInterfaceNamer(vmi *v1.VirtualMachineInstance) PodInterfaceNamer {
+	podInterfaceNamer := PodInterfaceNamer{}
+
+	podInterfaceNamer.mapDefaultNetworkToPodInterfaceName(vmi)
+	podInterfaceNamer.mapMultusNonDefaultNetworksToPodInterfaceName(vmi)
```

Comment:
I would inline those functions since they are not-exported and are not being reused, it's about 10 lines I think it's fine in terms of function length.
Not a must.

Reply:
I'm not thinking of lint here.
my purpose is to properly document what this map holds. 
Putting those implementation details on a function makes the code more readable.

Reply:
What I do not understand is the reason for these to be implemented as receivers.

Reply:
@EdDev, if I don't use them as receivers, he I'll need to pass the map to them as an input to build the map, like in here:
```
n[multusDefaultNetwork.Name] = PrimaryPodInterfaceName
```

Reply:
I may not have been clear.. I do not understand why we need an object and not direct functions to do the same.

Reply:
why not? is there a problem using that?

Reply:
IMO it is redundant "overhead" because the usage stays the same at the caller whether there is a type or not,
removing the type will simplify the implementation a bit and reduce noise.

Reply:
I have to agree with you both. change it to simpler implementation